code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -453,6 +453,32 @@
 	else
 		..()
 
+/datum/reagent/toxin/calciseptine //no, i'm not doing neurotoxin3
+	name = "calciseptine"
+	id = "calciseptine"
+	description = "A special kind of neurotoxin produced by snakes that will paralyse the victim after some time, and can not be metabolised without chems."
+	reagent_state = LIQUID
+	color = "#bfd16e"
+	metabolization_rate = 0
```

Comment:
That's crazy OP if players would manage to extract it and get it in a bottle, and trust me, they would.

Reply:
It's fairly difficult to extract. I know how, but it will take effort.

Reply:
actually, smoke grenades would have no effect, i'm going to make this only work on inject as it's a venom, not a poison.

Reply:
i'm not too worried, if a traitor manages to get their hands on this yeah there will be paralysis on the shuttle but they could have made a deathnade on the shuttle and it would have had better results

Reply:
You would probably just needed a person to soak up a lot of hits from this thing, grind it up, grind up the meat that has the reagents then replicated it with an odysseus,

Reply:
then i'll make it not work through inhaling or eating it (like all venoms are)

Reply:
Milking snakes is a bitch in real life, so having it be difficult here is appropriate. In addition, most snakevenoms are really difficult to aerosolize because extreme heat tends to denature them, as they tend to be peptides and proteins, rather than organophosphates.